README_BRANCHING.md

```diff
@@ -15,43 +15,48 @@ only be two branches that are relevant at all in the puppetlabs github repo:
 In the case of Puppet Server, the `stable` branch maps to the 1.x series of
 OSS Puppet Server releases.  These releases are compatible with Puppet 3.x.
 
-The `master` branch maps to the 2.x series of OSS Puppet Server releases.  These
-releases are compatible with Puppet 4.x.
+The `master` branch maps to the 2.1.x series of OSS Puppet Server releases.
```

Comment:
Maybe change this to "The `master` branch maps to the 2.x series of OSS Puppet Server releases.  The 2.0 release is compatible with Puppet 4.x.  The 2.1 release and later are intended to be compatible with Puppet 3.x and Puppet 4.x."